printer_mods/arhodiewithsoul/Bed/readme.md

```diff
@@ -0,0 +1,9 @@
+Contains updated Leadscrew Block and motor mounts for non-spec 185x185 (Prusa Mini), 200x200 (XYZPrinting DaVinci) build plates
+
+![a _leadscrew_block_202008_x2_001 (Large)](https://user-images.githubusercontent.com/80538348/116249173-7071b380-a764-11eb-854e-fd00b13e4fbe.jpg)
```

Comment:
For all images, please add them to a directory inside the mod (like `images/` for example) and reference those included images in the READMEs instead of uploading them githubusercontent.